torrent.go

```diff
@@ -2461,7 +2461,11 @@ func (t *Torrent) pieceRequestIndexOffset(piece pieceIndex) RequestIndex {
 }
 
 func (t *Torrent) updateComplete() {
+	before := t.Complete.Bool()
```

Comment:
I think there might be a more suitable place to put this, there should be a method that's only called when torrent completion transitions to complete from incomplete.

Reply:
sorry, I don't see such method.

Reply:
Ah I see the confusion. This flushes the entire torrent, when the entire torrent is complete. Do you want perhaps flush the individual memory-maps, when associated pieces become complete? That would be a more gradual approach.

Reply:
Sorry for not spotting this before. Similarly, if this was the approach, the Flush would move on to the piece implementation. Let me know your thoughts.